recipes/sagemaker-pyspark/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sagemaker_pyspark" %}
+{% set version = "1.2.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2f2fa0f8e8befe9253ff7c943f29313afb847a9ac8b7dd58187147a11bfe9478
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
```

Comment:
Are you sure all these dependencies are needed in the `host` section? Usually only `pip` and `python` should be in `host`, and the other dependencies in `run`.